assets/scss/03-organisms/_header-hamburger.scss

```diff
@@ -1,474 +1,739 @@
+
+            @include ma-button-reset;
+            position: relative;
+            display: block;
+            font-size: 1.25rem;
+            font-weight: 700;
+            letter-spacing: $letter-spacing-large;
+            line-height: 1.45;
+            padding: 15px 20px;
+            text-transform: uppercase;
+            transition: all 0.4s ease;
+            text-align: left;
+            background-color: $c-gray-lightest;
+            width: 100%;
+
+            &:focus {
+              background-color: $c-bd-divider;
+              outline: 2px solid $c-black;
+            }
+
+            .toggle-indicator {
+
+              @include ma-accordion-icon;
+            }
+
+            &[aria-expanded="true"] .toggle-indicator::after {
+              transform: rotate(135deg);
+            }
+
+            // Override the original styles.
+
+            & > svg {
+              position: relative;
+              top: auto;
+              left: auto;
+              transform: none;
+            }
+
+            & > span {
+              display: inline-block;
+            }
+          }
+
+          // Remove chevron.
+
+          &__link:after {
+            border: none;
+          }
+
+          a.direct-link {
+
+            @include ma-chevron;
+
+            &:after {
+              position: absolute;
+              right: 20px;
+              top: 25px;
+              border-right-color: $c-primary-alt;
+              border-bottom-color: $c-primary-alt;
+            }
+          }
+        }
+      }
+
+      // Override styles for desktop version utility nav content.
+
+      .show-menu & .ma__utility-nav__content {
+
+        @media ($bp-header-toggle-max) {
+
+          position: relative;
+          top: auto;
+          right: auto;
+          margin: 0;
+          padding: 0;
+          width: 100%;
+          color: $c-black;
+
+          height: 0;
+          opacity: 0;
+
+          transition: opacity 0.2s;
+          transition: height 0.6s;
+
+          .is-closed & {
+            transition: opacity 0.2s;
+          }
+
+          & .ma__utility-nav__content-title {
+            display: none;
+          }
+
+          & .ma__utility-nav__container {
+            padding: 0;
+            background-color: $c-gray-lightest;
+            overflow-y: visible;
+
+            .translated-rtl &,
+            .translated-ltr & {
+              height: auto;//calc(100vh - #{$header-mobile-controls-height * 2});
+            }
+          }
+
+          & .ma__utility-nav__container__close {// close window button in the dropdown.
+            display: none;
+          }
+
+          & .ma__utility-panel {
+            background-color: $c-gray-lightest;
+            color: $c-black;
+
+            @include ma-reset-list;
+          }
+
+          & .ma__utility-panel + .ma__utility-panel {
+            // margin-bottom: 1.75em; <-- original spacing inbetween. Remove this line once the design is set.
```

Comment:
Let's delete this comment if it's not needed

Reply:
The comment is removed.